R/plotScoreHeatmap.R

```diff
@@ -368,7 +370,7 @@ plotScoreHeatmap <- function(results, cells.use = NULL, labels.use = NULL,
         abs.max <- max(abs(range(scores, na.rm = TRUE)))
         breaks.len <- length(args$color)+1
         default_breaks <- seq(-abs.max, abs.max, length.out = breaks.len)
-        default_legend_breaks <- c(-abs.max, abs.max, length.out = 3)
+        default_legend_breaks <- c(-abs.max, abs.max)
```

Comment:
Oops, your bugfix here went the opposite way of what I'd have intended to do.  I would have left the length.out and swapped back to the original `seq()` had I caught it....

Ultimately though, :shrug:, new version just removes a middle label in the legend, which was always at zero.  **Now only the ends of the legend will be labeled but that's totally fine!**

Reply:
Ha, I was wondering which way it should go. Ended up just flipping a coin in my head.